bundle/config/resources.go

```diff
@@ -118,6 +118,12 @@ func (r *Resources) FindResourceByConfigKey(key string) (ConfigResource, error)
 		}
 	}
 
+	for k := range r.Experiments {
```

Comment:
Shall we just extend it to all supported resources instead of doing 1 by 1? You can rewrite this code to use `lib/dyn` to iterate over all existing resources and find the ones matching. So later when new resources are added they will automatically support `bind/unbind`

Reply:
I'd propose to do it after all resources are enabled in `bind/unbind` - the PRs still require individual integration tests to be introduced for each resource type

Reply:
That's fine too unless you have any ideas how maybe we can test bind for all the supported resources instead of writing individual tests?

Reply:
FWIW, I think doing it case by case is fine here. It forces thinking about the semantics for bind/unbind for specific resources. For example, I think that for dashboards it might not be as clear cut because of the workspace path / ID duality.